.bazelrc

```diff
@@ -0,0 +1 @@
+build --workspace_status_command $PWD/stamp.sh
```

Comment:
I think this should work without `$PWD/`, because Bazel runs the command specified here using bash in the root directory of the workspace. Maybe this also fixes the failure on macOS.

Reply:
The tests are only failing on [Windows](https://buildkite.com/bazel/bazelisk/builds/134#29116b83-4a05-4ec3-bc8b-d0962745ffd7).

Do you know of any good ways to create cross-platform `--workspace_status_command` scripts? I've seen that a few projects are using Python, but that is very overkill for what I'd like to do here.

Maybe we should just skip the `.bazelrc`? Adding the workspace data is only really needed when building release artefacts, which I'm assuming is done manually today.

Reply:
Ah, true :D I looked in the wrong spot.

I'm also not aware of a better mechanism that works cross-platform :neutral_face: 

We can drop the `.bazelrc` file and I'll manually specify that flag when building the release artifacts then. Thanks!

Reply:
I dropped it for now. :)